docs/source/index.rst

```diff
@@ -34,6 +32,9 @@ In case you are already fluent in compiling C++ projects and HPC, running PIC si
 .. _wiki: https://github.com/ComputationalRadiationPhysics/picongpu/wiki
 .. _official homepage: http://picongpu.hzdr.de
 
+************
+Installation
```

Comment:
Necessary since I can currently not set the caption of a toctree as chapter title